### PR TITLE
Update chef to 18.1

### PR DESCRIPTION
**Pull request by nikhil2611**
## Description
Bundling workstation with chef 18.1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: src/workstation-gui/Gemfile.lock [hunk 1]
```diff
@@ -206,7 +206,7 @@ GEM
     chef-telemetry (1.1.1)
       chef-config
       concurrent-ruby (~> 1.0)
-    chef-utils (18.0.185)
+    chef-utils (18.1.0)
       concurrent-ruby
     chef-vault (4.1.10)
```
Comment (Tyrael) [inline src/workstation-gui/Gemfile.lock:211]: shouldn't this one also bumped to 4.1.11?

Comment (vkarve-chef) [reply src/workstation-gui/Gemfile.lock:211]: with rebase, we'll automatically have it when this PR is merged.

Comment (Tyrael) [reply src/workstation-gui/Gemfile.lock:211]: I see, thanks!

**Comment by nikhil2611** [comment]
Merged in different PR: https://github.com/chef/chef-workstation/pull/3093

**Pull request closed without merging**